Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -296,6 +299,28 @@ void HoI4::Country::convertRelations(const CountryMapper& countryMap)
 }
 
 
+void HoI4::Country::convertStrategies(const CountryMapper& countryMap)
+{
+	auto srcStrategies = sourceCountry.getAI()->getStrategies();
```

Comment:
unless you're modifying srcStrategies or strategyType below, keeping it a const auto& means no copying is done and whole thing is faster.